kernel/riscv64/KERNEL.RISCV64_ZVL128B

```diff
@@ -256,7 +256,7 @@ ifndef ZGEMM_BETA
 ZGEMM_BETA = zgemm_beta_rvv.c
 endif
 
-ifeq ($(BUILD_BFLOAT16), 1)
+ifeq ($(BUILD_HFLOAT16), 1)
```

Comment:
This fixes a bug from the original RISCV64 SHGEMM MR.

Reply:
[https://github.com/OpenMathLib/OpenBLAS/pull/5290](https://github.com/OpenMathLib/OpenBLAS/pull/5290)